relayer/workers/parachaincommitmentrelayer/main.go

```diff
@@ -10,15 +10,18 @@ import (
 	"github.com/snowfork/polkadot-ethereum/relayer/chain"
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/ethereum"
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/relaychain"
 	"github.com/snowfork/polkadot-ethereum/relayer/contracts/inbound"
 	"github.com/snowfork/polkadot-ethereum/relayer/crypto/secp256k1"
 	"github.com/snowfork/polkadot-ethereum/relayer/substrate"
 )
 
 type Worker struct {
 	parachainConfig             *parachain.Config
+	relaychainConfig            *relaychain.Config
 	ethereumConfig              *ethereum.Config
 	parachainConn               *parachain.Connection
+	relaychainConn              *relaychain.Connection
```

Comment:
Should `relaychainConn`/`relaychainConfig` be added to this worker in a separate PR?

Reply:
yep, sorry should have done in different PR, gonna leave it in for now as i'll be working on follow up PR today